src/app/api/_api-services/redis_service/index.ts

```
@@ -580,4 +590,94 @@ export class RedisService {
+		await this.client.set(key, JSON.stringify(data), 'EX', SIX_HOURS_IN_SECONDS);
+	}
+
+	static async GetTurnoutPercentageAnalytics(network: ENetwork): Promise<ITurnoutPercentageData | null> {
+		const key = `turnout_percentage_analytics:${network}`;
+		const cachedData = await this.client.get(key);
+		return cachedData ? JSON.parse(cachedData) : null;
+	}
+
+	static async SetTurnoutPercentageAnalytics({ network, data }: { network: ENetwork; data: ITurnoutPercentageData }): Promise<void> {
+		const key = `turnout_percentage_analytics:${network}`;
+		await this.client.set(key, JSON.stringify(data), 'EX', 21600); // Cache for 6 hours
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)